python.d/smartd_log.chart.py

```diff
@@ -198,11 +198,10 @@ def create_lines(attrid):
             return result
 
         # Use configured attributes, if present. If something goes wrong we don't care.
-        if self.attr:
-            try:
-                ORDER = [attr for attr in self.attr.split() if attr in SMART_ATTR.keys()]
-            except Exception:
-                pass
+        try:
+            ORDER = [attr for attr in self.attr.split() if attr in SMART_ATTR.keys()] or ORDER
+        except Exception:
+            pass
         self.order = [''.join(['attrid', i]) for i in ORDER]
```

Comment:
Sorry @pkoenig10. My bad. This will not work (thx to codacy) 😄
do something like
```python
        try:
            custom_order = [attr for attr in self.attr.split() if attr in SMART_ATTR]
        except AttributeError:
            custom_order = None
        self.order = [''.join(['attrid', i]) for i in custom_order or ORDER]
```
or whatever you prefer